assembly/std/coins/transfer.ts

```diff
@@ -0,0 +1,39 @@
+import { env } from '../../env';
+import { Address } from '../address';
+
+/**
+ * Transfers coins from the current address 'to' a given address.
+ *
+ * @param to - the address to send coins to.
+ * @param amount - value in the smallest unit.
+ * @see [massa units standard](https://github.com/massalabs/massa-standards/blob/main/units.md)
+ *
+ * @throws
+ * - if the given address is not a valid address.
+ * - if the balance of the current address is insufficient to make the transaction.
+ *
+ */
+export function transferCoins(to: Address, amount: u64): void {
```

Comment:
I propose to name it transfer as we are in the coins package:
```suggestion
export function transfer(to: Address, amount: u64): void {
```

Reply:
To be done in another issue.